roachtest: npgsql failed

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12162079?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12162079?buildTab=artifacts#/npgsql) on release-23.1 @ [1d5f07d043035d14735c328a59384cd96cb47b09](https://github.com/cockroachdb/cockroach/commits/1d5f07d043035d14735c328a59384cd96cb47b09):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.12-dev-1d5f07d043035d14735c328a59384cd96cb47b09
Tests run against npgsql v7.0.2
3284 Total Tests Run
2577 tests passed
707 tests failed
248 tests skipped
57 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CommandTests(Multiplexing).SingleQuery -  (unexpected)
For a full summary look at the npgsql artifacts 
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12162079/npgsql/1697206926060/1697207390992)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-32355

--- a/pkg/cmd/roachtest/tests/npgsql_blocklist.go
+++ b/pkg/cmd/roachtest/tests/npgsql_blocklist.go
@@ -717,6 +717,7 @@ var npgsqlIgnoreList = blocklist{
 	`Npgsql.Tests.CommandTests(Multiplexing).Cursor_move_RecordsAffected `:                            "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).QueryNonQuery`:                                           "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).SingleNonQuery`:                                          "flaky",
+	`Npgsql.Tests.CommandTests(Multiplexing).SingleQuery`:                                             "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).Statement_mapped_output_parameters(Default)`:             "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).Use_across_connection_change(NotPrepared)`:               "flaky",
 	`Npgsql.Tests.CommandTests(NonMultiplexing).Cached_command_clears_parameters_placeholder_type`:    "flaky",

--- a/pkg/cmd/roachtest/tests/ruby_pg_blocklist.go
+++ b/pkg/cmd/roachtest/tests/ruby_pg_blocklist.go
@@ -200,5 +200,6 @@ var rubyPGBlocklist = blocklist{
 }
 
 var rubyPGIgnorelist = blocklist{
-	`running with sync_* methods PG::Connection consume_input should raise ConnectionBad for a closed connection`: "unknown",
+	`PG::Connection OS thread support Connection.new shouldn't block a second thread`:                             "flaky",
+	`running with sync_* methods PG::Connection consume_input should raise ConnectionBad for a closed connection`: "flaky",
 }

